firebase-perf/dev-app/src/main/AndroidManifest.xml

```diff
@@ -38,6 +38,9 @@
         <meta-data
             android:name="fragment_sampling_percentage"
             android:value="100.0" />
+        <meta-data
```

Comment:
Can you also update the e2e test app so that it will generate this trace periodically? but I think our e2e test app does not generate app start trace. If so, it is out of the scope of this PR.

Reply:
Checked just now and you are right, our E2E test app doesn't generate `_app_start` at all.

Reply:
IIRC the test skips the onResume() callback